templates/emails/transactional/unsubscribe-newsletter.txt

```diff
@@ -1,12 +1,7 @@
+{% extends "base.txt.jinja2" %}
+
+{% block content %}
 You have been unsubscribed from SimpleLogin newsletter.
 
 We would love to know why do you want to unsubscribe from our communication.
-
-Please let us know by replying to this email.
-
-Best,
-SimpleLogin Team.
-
-
-
-
+{% endblock %}
```

Comment:
Nits: as user can't reply to this email anymore, we can remove the "We would love to know why do you want to unsubscribe from our communication." sentence.

Reply:
Removed.